error.go

```diff
@@ -92,6 +92,12 @@ func (*ErrReturn) typ() typErr {
 	return TypeErrReturn
 }
 
+type ErrDummy struct{}
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported type ErrDummy should have comment or be unexported